### PR TITLE
[18.06] bump ver 18.06.3

### PR DESCRIPTION
**Pull request by andrewhsu**

### CODE REVIEW

**Comment by thaJeztah** [comment]
full diff: https://github.com/docker/docker-ce/compare/v18.06.2-ce...4167aa511bd25210b679a9f53787d846e645fa37

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 For more information on the list of deprecated flags and APIs, have a look at
 https://docs.docker.com/engine/deprecated/ where you can find the target removal dates 
 
+## 18.06.3-ce (2019-02-19)
+
```
Comment (thaJeztah) [inline CHANGELOG.md]: Do we want to mention the packaging-fix in this release?

Comment (andrewhsu) [reply CHANGELOG.md]: yes, sorry this is PR is WIP. i'll add words.

**Comment by thaJeztah** [comment]
oh, right, so we'll need https://github.com/docker/engine/pull/156 to be pulled in as well first I guess?

**Comment by andrewhsu** [comment]
@thaJeztah stuff is now more up to date: https://github.com/docker/docker-ce/compare/v18.06.2-ce...18.06

and i've added words

**Pull request merged**